### ISSUE TITLE
Why don't you upload lpython to PyPI?

### ISSUE THREAD

**Issue by Foxchandaisuki** [lcompilers/lpython#2674]
It would be good if you could upload lpython to PyPI so that we can install it via pip. This would make it more convenient for people to install lpython without having to install Anaconda.

**Comment by kmr-srbh** [comment]
Hello @Foxchandaisuki! LPython is not a Python package, but a compiler built using LLVM. It aims to be a subset of Python and compiled by nature. It cannot be uploaded to PyPi.

If you are facing issues with the build process, please let us know. We are more than happy to help you!

**Comment by Foxchandaisuki** [comment]
> Hello @Foxchandaisuki! LPython is not a Python package, but a compiler built using LLVM. It aims to be a subset of Python and compiled by nature. It cannot be uploaded to PyPi.
> 
> If you are facing issues with the build process, please let us know. We are more than happy to help you!

@kmr-srbh Thank you for answer. Then do you have a plan to make LPython a standalone implementation that doesn't require Anaconda to be installed, like CPython?

**Comment by certik** [comment]
We can't upload until this issue is resolved: https://github.com/pypi/support/issues/3204. I just pinged the PyPi admins again.

**Comment by kmr-srbh** [comment]
> We can't upload until this issue is resolved: [pypi/support#3204](https://github.com/pypi/support/issues/3204). I just pinged the PyPi admins again.

Thanks for this @certik! I did not know that software binaries could be packaged to PyPi. Please pardon me for the mistake.

**Comment by carlkl** [comment]
why not use a pypi name like `lcompiler` or `lcompilerpython` or `lcpython`?

BTW: there is a pypi project named [lpython-emulation](https://pypi.org/project/lpython-emulation)

**Comment by Shaikh-Ubaid** [comment]
>BTW: there is a pypi project named [lpython-emulation](https://pypi.org/project/lpython-emulation)

Yes, there is. It is maintained by me. We have our `src/runtime/lpython.py` (which contains the typing utilities) deployed there.

**Comment by carlkl** [comment]
So `lpython-compiler` could be a suitable name I guess.
